models/quality_measures/quality_measures_models.yml

```diff
@@ -131,6 +131,57 @@ models:
       - name: tuva_last_run
         description: The date and timestamp of the dbt run.
 
+## Shared Exclusions
+  - name: shared_exclusions__frailty
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}{{var('tuva_schema_prefix')}}_quality_measures{% else %}quality_measures{%- endif -%}
+      alias: _frailty
+      tags: shared_exclusions
```

Comment:
The tag for all models should be the same as the data mart, `quality_measures`. This allows us to run this data mart only.

Reply:
All of the tags for shared exclusion models have been changed to `quality_measures`